.github/workflows/ci.yaml

```diff
@@ -0,0 +1,127 @@
+name: CI
+
+on: [pull_request]
+
+jobs:
+  structure_test_at_workspace_root:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v1
+      - name: Mount bazel cache
+        uses: actions/cache@v2
+        with:
+          path: "/home/runner/.cache/bazel"
+          key: bazel-structure_test_at_workspace_root
+      - name: Install bazelisk
+        run: |
+          curl -LO "https://github.com/bazelbuild/bazelisk/releases/download/v1.7.5/bazelisk-linux-amd64"
+          mkdir -p "${GITHUB_WORKSPACE}/bin/"
+          mv bazelisk-linux-amd64 "${GITHUB_WORKSPACE}/bin/bazel"
+          chmod +x "${GITHUB_WORKSPACE}/bin/bazel"
+      - name: Test
+        run: |
+          "${GITHUB_WORKSPACE}/bin/bazel" test \
+            --config=github \
+            //:structure_test_at_workspace_root
+  tests_slash_docker:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v1
+      - name: Mount bazel cache
+        uses: actions/cache@v2
+        with:
+          path: "/home/runner/.cache/bazel"
+          key: bazel-tests_slash_docker
+      - name: Install bazelisk
+        run: |
+          curl -LO "https://github.com/bazelbuild/bazelisk/releases/download/v1.7.5/bazelisk-linux-amd64"
+          mkdir -p "${GITHUB_WORKSPACE}/bin/"
+          mv bazelisk-linux-amd64 "${GITHUB_WORKSPACE}/bin/bazel"
+          chmod +x "${GITHUB_WORKSPACE}/bin/bazel"
+      - name: Test
+        run: |
+          "${GITHUB_WORKSPACE}/bin/bazel" test \
+            --config=github \
+            //tests/docker/package_managers:download_pkgs_run_test \
+            //tests/docker/package_managers:download_pkgs_with_additional_repos_run_test \
+            //tests/docker/package_managers:gpg_test \
+            //tests/docker/toolchain_container:test-rbe-test-xenial-with-pkgs \
+            //tests/docker/util:test_extracted_file \
+            //tests/docker/util:test_container_commit_metadata
+  tests_slash_container:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v1
+      - name: Mount bazel cache
+        uses: actions/cache@v2
+        with:
+          path: "/home/runner/.cache/bazel"
+          key: bazel-tests_slash_container
+      - name: Install bazelisk
+        run: |
+          curl -LO "https://github.com/bazelbuild/bazelisk/releases/download/v1.7.5/bazelisk-linux-amd64"
+          mkdir -p "${GITHUB_WORKSPACE}/bin/"
+          mv bazelisk-linux-amd64 "${GITHUB_WORKSPACE}/bin/bazel"
+          chmod +x "${GITHUB_WORKSPACE}/bin/bazel"
+      - name: Test
+        run: |
+          "${GITHUB_WORKSPACE}/bin/bazel" test \
```

Comment:
Would it make sense to group some of this into a non-yamly thing? I've had very good experiences with [test_suite](https://docs.bazel.build/versions/master/be/general.html#test_suite) in the past. If it's annoying to do in this PR we can leave it as is until it becomes annoying to keep in sync.

Reply:
test_suite is a good idea!

Reply:
I'm also contemplating a tool to generate the YAML here.  Looking at the presubmit and trying to see what's overlapping.